tests/acceptance/acceptancesuite.js

```diff
@@ -59,13 +60,19 @@ test(`Facets load on the page, and can affect the search`, async t => {
   const facets = FacetsPage.getFacetsComponent();
   const filterBox = facets.getFilterBox();
 
+  const getPageUrl = ClientFunction(() => window.location.href);
```

Comment:
I'm not sure this should be part of an acceptance test. I think we've been kinda blurring the line between our unit and acceptance tests. Part of the reason for that, I think, is that it can be hard for us to write unit tests for our components. A lot of things need to be mocked that are difficult to mock. 

I think we should try and make this into a unit test of FilterOptions, if possible. Essentially, what we want to test is that if storeOnChange is false, we don't call persistentStorage.set until the apply button is also clicked. With acceptance tests, we want to be testing normal customer flows that span multiple pages and components. This particular test is scoped to just the FilterOptions component. 

It seems like you may have already added unit test coverage for this. If so, I think we can just remove your changes from this file.

Reply:
Yep, I do have unit test coverage so I'll remove the acceptance test changes